rules/eslint/best-practices/walmart.js

```diff
@@ -44,8 +44,12 @@ module.exports = {
     "no-fallthrough": 2,
     // disallow the use of leading or trailing decimal points in numeric literals
     "no-floating-decimal": 0,
+    // disallow the type conversions with shorter notations
+    "no-implicit-coercion": 0,
     // disallow use of eval()-like methods
     "no-implied-eval": 2,
+    // disallow this keywords outside of classes or class-like objects
+    "no-invalid-this": 0,
```

Comment:
Yes please! Only case need to disable is wrapper for passing in window or global based on browser v node and that's a single case...